Yoast/YoastSEO.js
# Add french function words

## Summary

This PR can be summarized in the following changelog entry: Adds prominent words for French, props Sylvain Perret and Vianney Andre.  

## Test instructions

This PR can be tested by following these steps:

* Run `grunt build:js`.
* Use the prominent word example in your browser.
* Set language to fr_FR.
* Paste a French text into the example.
* Make sure you see no strange word combinations, counts, etc.

Fixes #1165
